terraform/modules/s3_bucket/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "bucket_name" {
+  type = string
+}
+
+variable "access_log_bucket_name" {
+  type = string
+}
+
+variable "kms_key_arn" {
+  type        = string
+  default     = null
+  description = "Optional. KMS key to encrypt bucket and access logs bucket."
+}
+
+variable "force_destroy" {
+  description = "Allow the buckets to be destroyed by Terraform even if they are not empty."
+  default     = false
+}
+
+variable "restrict_public_buckets" {

```

Comment:
do we know that we'll need this to be configurable? 

This whole s3 encapsulation is quite generic, which I don't generally think is worth applying YAGNI to, but this one in particular feels like a dangerous option to give people, and if we never expect to need it I'd vote for removing the option.

Reply:
That's fair - I don't _think_ we're going to need any public buckets, the only data that will be shared between accounts should hopefully be managed by AWS backup, so I'll remove this